### DISCUSSION TITLE
Cup Profile... Cupper's Correction

### DISCUSSION THREAD

**Discussion by DennieSch** [artisan-roaster-scope/artisan#1690] [General]
Might there be a way to directly add 'points' to the cupping score that isn't 'factored or proportioned' into/with the other cupping elements. I can't seem to find a way to do this. 
The reason I ask is, I'm trying to capture/input the cupping score that Sweet Maria provides with of their coffees. Their approach to scoring includes a 'Cupper's Correction' which is directly added to the cupping score. (So, I'm not cupping my own roast results but just trying to capture the information provided by Sweet Maria.)
I'm not sure this is currently a 'feature' on the Cup Profile window.

Cheers,
Dennie

**Comment by MAKOMO** [comment]
just added

**Reply by DennieSch** [reply]
Tried it and it works great. Thank you for the quick response!

**Comment by DennieSch** [comment]
Thanks again for adding the 'Cupper's Correction'. At the risk of being a 'pest', would it be possible to add this (when applicable) to the Cup Profile image? That way it will show up when generating an image of the cup profile and generating a Roast Report. 

Here is an example of how Sweet Maria's handles the 'presentation' of this...

![Cuppers correction at SM](https://github.com/user-attachments/assets/950ab3c9-0118-4349-a9dc-d682cf9f900c)

**Reply by MAKOMO** [reply]
Dennie, would this work?

<img width="649" alt="Screenshot 2024-11-11 at 16 27 04" src="https://github.com/user-attachments/assets/a6e7792a-6b5d-4b3e-ab56-bfaf850feede">

Its part of the Continuous Build which is generated at this moment.

**Reply by DennieSch** [reply]
That's perfect Marko! Thank you. I tried the CB and works great. One last thought (a.k.a. request 😁), any chance this could be added as one of the drop-downs in the "Stats Summary".
Cheers,
Dennie

**Reply by MAKOMO** [reply]
What exactly are you referring to by "it" here? The Correction value?

**Reply by DennieSch** [reply]
Yes, just the Correction value. 
I have the Cupping Score in my Stats Summary and would like to show the Correction value underneath it. 
If it could show up as an option/selection (as shown below) that would be great. 
![Screenshot 2024-11-11 114537](https://github.com/user-attachments/assets/1b384021-fa96-4358-ae1d-9272d01be0f6)

**Reply by DennieSch** [reply]
Hi Marko- Just following up on this and to make sure I answered your question. Thanks!
Cheers... Dennie

**Reply by MAKOMO** [reply]
Dannie, this was added and will be part of Fridays v3.1 release.

**Reply by DennieSch** [reply]
👍 👍
